draft-ietf-httpbis-message-signatures.md

```diff
@@ -183,14 +180,14 @@ An HTTP header field value is canonicalized as follows:
 
 This section contains non-normative examples of canonicalized values for header fields, given the following example HTTP message:
 
-~~~
+~~~ http-message
 HTTP/1.1 200 OK
 Server: www.example.com
 Date: Tue, 07 Jun 2014 20:51:35 GMT
-X-OWS-Header:   Leading and trailing whitespace.   
-X-Obs-Fold-Header: Obsolete  
+X-OWS-Header:   Leading and trailing whitespace.
```

Comment:
```suggestion
X-OWS-Header:   Leading and trailing whitespace.    
```

@richanna @mnot are those whitespaces needed? The reader won't probably see them :)

Reply:
The trailing whitespace on `X-OWS-Header` is important, as it is intended to demonstrate that trailing whitespace is stripped during canonicalization of a header field. I would think trailing whitespace in a codeblock should be preserved. What processor is stripping them out? `kramdown-rfc2629` or `xml2rfc`?

Reply:
Oh, no that's just EditorConfig. Do you think people will actually check for trailing whitespace in the RFC?

Reply:
If someone copies/pastes the example, they should get what the example is intending to demonstrate. Maybe not super critical in this non-normative example, but I'd like to add a set of canonicalization test cases at some point, where maintaining that trailing whitespace will be very important.

Reply:
OK, cool. Will restore and remember not to edit this file with editorconfig enabled; perhaps we should exempt it in `.editorconfig`.